baseplate/clients/requests.py

```diff
@@ -101,6 +107,42 @@ def http_adapter_from_config(
     return ValidatingHTTPAdapter(**kwargs)
 
 
+PROM_NAMESPACE = "http_client"
+
+# Latency histogram of HTTP calls made by clients
+# buckets are defined above (from 100µs to ~14.9s)
+LATENCY_SECONDS = Histogram(
+    f"{PROM_NAMESPACE}_latency_seconds",
+    "Latency histogram of HTTP calls made by clients",
+    [
+        "http_method",
+        "http_success",
+        "http_slug",
+    ],
+    buckets=default_latency_buckets,
+)
+# Counter counting total HTTP requests started by a given client
+REQUESTS_TOTAL = Counter(
+    f"{PROM_NAMESPACE}_requests_total",
+    "Total number of HTTP requests started by a given client",
+    [
+        "http_method",
+        "http_success",
+        "http_response_code",
+        "http_slug",
```

Comment:
I have an open PR for renaming this to client_name since slug doesn't really tell me what it's an identifier for

Reply:
[PR](https://github.snooguts.net/reddit/baseplate.spec/pull/46) has been merged, this should change to `http_client_name`